src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/SanitizerInfo.cs

```diff
@@ -38,7 +43,7 @@ public SanitizerInfo(string fullTypeName, bool isInterface, bool isConstructorSa
         /// <summary>
         /// Methods that untaint tainted data.
         /// </summary>
```

Comment:
Copy and modify the &lt;remarks&gt; from `SourceInfo.TransferMethods`' doc comments.